systemjs/react-hello-world/README.md

```diff
@@ -0,0 +1,13 @@
+# react-hello-world
+
+The react-hello-world example demonstrates the following:
+
+- Loading React and ReactDOM from a CDN. This is accomplished with an import map and allows for multiple modules to share the same React instance.
+- Using [Babel CLI](https://babeljs.io/docs/en/babel-cli) to compile a file to [System.register format](https://github.com/systemjs/systemjs/blob/master/docs/system-register.md). This is done with the [`@babel/plugin-transform-modules-systemjs`](https://babeljs.io/docs/en/babel-plugin-transform-modules-systemjs) plugin.
+- Loading javascript modules with `<script type="systemjs-module" src="./path-to-file.js">`.
+
+This example shows foundational principles, but is not a comprehensive starter kit for SystemJS and React.
```

Comment:
In due course, we should probably have a "next steps" section for each providing some branching off points. Eg for this we might want to link to a more comprehensive React example, as well as a Babel with optimization via Webpack or Rollup example.

Reply:
Agreed 👍